src/main/java/org/betonquest/betonquest/utils/location/AbstractData.java

```diff
@@ -115,13 +115,13 @@ public T get(final Profile profile) throws QuestRuntimeException {
     }
 
     private T parseVariableObject(final Profile profile) throws QuestRuntimeException {
-        if (profile == null) {
-            throw new QuestRuntimeException("Variable cannot be accessed without the player."
-                    + " Consider changing it to absolute coordinates");
-        }
         final String[] variables = new String[this.objectVariables.size()];
         for (int i = 0; i < this.objectVariables.size(); i++) {
             final Variable var = this.objectVariables.get(i);
+            if (profile == null && !var.isStaticness()) {
+                throw new QuestRuntimeException("Variable cannot be accessed without the player."
```

Comment:
Is it possible to give more context here? It might get really hard to find the faulty variable if you don't name it and the quest package.